source/WorkingPractices/macros.rst

```diff
@@ -114,6 +118,8 @@ To add upgrade macros to LFRic the following steps can be followed:
 
 .. code-block::
 
+    export CYLC_VERSION=8 # (if required)
```

Comment:
```suggestion
    export CYLC_VERSION=8 
```

I'd suggest that "if required" just confuses things, leave the user to know if they've got cylc 8 loaded, otherwise it is required :)